docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/index.rst

```diff
@@ -0,0 +1,83 @@
+.. _2019_hemoglobin_anemia_and_iron_deficiency:
+
+=======================================
+Hemoglobin, Anemia, and Iron Deficiency
+=======================================
+
+.. toctree::
+   :maxdepth: 1
+   :glob:
+
+   *
+   */index
```

Comment:
This structure is slightly confusing to me, as I usually expect to see the contents for the current page at the top (i.e. with a `.. contents:: :local:` directive), but I think it's also useful to have the links to all the subpages in one place near the top.  Maybe put the local contents at the top and put this `toctree` at the end of the Topic Overview section?